narf/combineutils.py

```diff
@@ -121,6 +116,20 @@ def __init__(self, filename):
             self.npoly2dreggroups = len(self.poly2dreggroups)
             self.nnoigroups = len(self.noigroups)
 
+            # reference meta data if available
+            self.metadata = {}
+            if "meta" in f.keys():
+                from narf.ioutils import pickle_load_h5py
+                self.metadata = pickle_load_h5py(f["meta"])
+                self.channel_axes = self.metadata["channel_axes"]
+            else:
+                self.channel_axes = {
+                    "ch1": [hist.axis.Integer(0, self.nbins, underflow=False, overflow=False, name="obs")]
+                }
+                self.channel_axes["ch2"] = [hist.axis.Integer(0, self.nbinsmasked, underflow=False, overflow=False, name="masked")]
```

Comment:
This should probably have "masked" in the name, and also I think this needs to be protected against the case where there are no masked bins (ie just skip creating the second channel in that case)